package.json

```diff
@@ -1,29 +1,29 @@
 {
   "name": "datahub-link-generator",
-  "version": "3.0.0",
-  "description": "",
+  "version": "1.0.0",
+  "description": "A browser extension to generate DataHub links directly from GitHub",
   "private": true,
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/berkeley-dsep-infra/nbgitpuller-link-generator-webextension.git"
+    "url": "git+https://github.com/yuvipanda/nbgitpuller-link-generator.git"
```

Comment:
This is the wrong org. Given this change and the one above, was this PR based on an old commit?

Reply:
Not sure, how this link got picked up. I have fixed that as part of my latest commit.